include/functions.h

```diff
@@ -1122,41 +1122,41 @@ Gfx* Gfx_SetFog(Gfx* gfx, s32 r, s32 g, s32 b, s32 a, s32 near, s32 far);
 Gfx* Gfx_SetFogWithSync(Gfx* gfx, s32 r, s32 g, s32 b, s32 a, s32 near, s32 far);
 Gfx* Gfx_SetFog2(Gfx* gfx, s32 r, s32 g, s32 b, s32 a, s32 near, s32 far);
 Gfx* Gfx_CallSetupDL(Gfx* gfx, u32 i);
```

Comment:
Should we align the `Gfx_CallSetupDL*` names with these new names?
I originally went for a more explicit approach with `Call` (ie. what `gSPDisplayList` ends up doing), but perhaps it's not necessary. Or we could add `Call` to the new names, not sure what's better

Reply:
I think just `SetupDL` would work?

Reply:
yea seems fine

Reply:
It would probably work.
My main concern is `SetupDL` sounds like the action of "setting up a DL" as opposed to "calling/applying a setup DL", and that might be confusing if you aren't familiar with the system (especially since these functions are usually called in the context of setting up one or multiple object DLs). But I don't know if it really matters

Reply:
Well to me a setup DL sets things up for subsequent DLs, so in my mind that works

Reply:
Yeah it's probably fine, we can go with that.
Also, just so we don't forget, there is `Gfx_CallSetupDLImpl` and `Gfx_CallSetupDLAtPtr` to rename as well